gulpfile.js

```diff
@@ -25,7 +25,7 @@ gulp.task('lint', ['eslint']);
 gulp.task('default', ['lint']);
 
 gulp.task('gen-prefs', function(cb){
-  var contents = '// Set prefs to use a local content server\n';
+  var contents = '// Set prefs to use a local content server\n'; // eslint-disable-line
```

Comment:
@6a68 Beats me! I was trying to debug into it, and I remember hitting the same thing with Grunt in a separate repo where it didn't really like certain ES6-isms. I think we solved it in the universal-search-content repo since that repo was using Babel and we could name the file gulpfile.babel.js to get some nice ES6 magic.

But for now it looks like this gulpfile.js is a weird exception to the rule(s).